<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFeedDialog</class>
 <widget class="QDialog" name="AddFeedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create new feed</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../retroshare-gui/src/gui/images.qrc">
    <normaloff>:/images/rstray3.png</normaloff>:/images/rstray3.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="headerFrame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>64</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#headerFrame{background-image: url(:/images/connect/connectFriendBanner.png);}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="headerIcon">
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="FeedReader_images.qrc">:/images/FeedReader.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="headerLabel">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Arial'; font-size:24pt; font-weight:600; color:#ffffff;&quot;&gt;Feed Details&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="7" column="0">
       <widget class="QGroupBox" name="authenticationGroupBox">
        <property name="title">
         <string>Authentication (not yet supported)</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="useAuthenticationCheckBox">
           <property name="text">
            <string>Feed needs authentication</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="userLabel">
           <property name="text">
            <string>User</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="passwordLabel">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="userLineEdit"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="passwordLineEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QGroupBox" name="updateInteralGroupBox">
        <property name="title">
         <string>Update interval</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="useStandardUpdateInterval">
           <property name="text">
            <string>Use standard update interval</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="updateIntervalLabel">
           <property name="text">
            <string>Interval in minutes (0 = manual)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="updateIntervalSpinBox">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <layout class="QHBoxLayout" name="lastUpdateLayout">
           <item>
            <widget class="QLabel" name="lastUpdateLabel">
             <property name="text">
              <string>Last update</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lastUpdate">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Never</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QGroupBox" name="storageTimeGroupBox">
        <property name="title">
         <string>Storage time</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="useStandardStorageTimeCheckBox">
           <property name="text">
            <string>Use standard storage time</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="storageTimeLabel">
           <property name="text">
            <string>Days (0 = off)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="storageTimeSpinBox">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QGroupBox" name="proxyGroupBox">
        <property name="title">
         <string>Proxy</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="useStandardProxyCheckBox">
           <property name="text">
            <string>Use standard proxy</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="serverLabel">
           <property name="text">
            <string>Server</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="proxyAddressLineEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="portLabel">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="proxyPortSpinBox">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="previewButton">
          <property name="text">
           <string>Preview</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="typeGroupBox">
        <property name="title">
         <string>Type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="margin">
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="typeForumRadio">
             <property name="text">
              <string>Forum</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="forumComboBox"/>
           </item>
           <item>
            <widget class="QLabel" name="forumNameLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true">Forum name</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="typeLocalRadio">
           <property name="text">
            <string>Local Feed</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QGroupBox" name="flagGroupBox">
        <property name="title">
         <string>Misc</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="activatedCheckBox">
           <property name="text">
            <string>Activated</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="useInfoFromFeedCheckBox">
           <property name="text">
            <string>Use name and description from feed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="updateForumInfoCheckBox">
           <property name="text">
            <string>Update forum information</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="embedImagesCheckBox">
           <property name="text">
            <string>Embed images (experimental for local feeds)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="saveCompletePageCheckBox">
           <property name="text">
            <string>Save complete web page (experimental for local feeds)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QVBoxLayout" name="descriptionLayout">
        <item>
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="descriptionPlainTextEdit"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="nameLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="urlLabel">
          <property name="text">
           <string>RSS-Feed-URL:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="urlLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="nameLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>urlLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>descriptionPlainTextEdit</tabstop>
  <tabstop>typeForumRadio</tabstop>
  <tabstop>forumComboBox</tabstop>
  <tabstop>activatedCheckBox</tabstop>
  <tabstop>useInfoFromFeedCheckBox</tabstop>
  <tabstop>updateForumInfoCheckBox</tabstop>
  <tabstop>useAuthenticationCheckBox</tabstop>
  <tabstop>userLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>useStandardStorageTimeCheckBox</tabstop>
  <tabstop>storageTimeSpinBox</tabstop>
  <tabstop>useStandardUpdateInterval</tabstop>
  <tabstop>updateIntervalSpinBox</tabstop>
  <tabstop>useStandardProxyCheckBox</tabstop>
  <tabstop>proxyAddressLineEdit</tabstop>
  <tabstop>proxyPortSpinBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="FeedReader_images.qrc"/>
  <include location="../../../retroshare-gui/src/gui/images.qrc"/>
 </resources>
 <connections/>
</ui>
